documentation/sphinx/source/release-notes/release-notes-7135.rst

```diff
@@ -0,0 +1,71 @@
+.. _release-notes:
+
+#############
+Release Notes
+#############
+
+71.3.5
+======
+
+Features
+--------
+
+Performance
+-----------
+
+Reliability
+-----------
+
+* Ensure exponential backoff while refreshing KMS URLs.
+* Update EncryptKeyProxy KMS health check timeouts.
+* Avoid commit proxy and storage server commit timeout due to KMS failure.
+
+
+Fixes
+-----
+
+* Fix quota throttler clear cost estimation.
+* Fix the DataLossRecovery workload stuck issue.
+* Wait file deleted in blob manifest cleanup.
+* Fixed slowtask and OOM in large blob granule purges, and improved observability.
+* Improve Blob Manager’s granule balancing algorithm in pathological cases.
+* Allow granule purge to continue when the cluster is unhealthy.
+* Restore original function names for old blob granule API.
+
+Status
+------
+
+Bindings
+--------
+
+Other Changes
+-------------
+
+* Report how many logical ranges and shards have been over-replicated by custom replication.
+
+
+Change Log
+---------------------
+
+* https://github.com/apple/foundationdb/pull/10664
+* https://github.com/apple/foundationdb/pull/10811
+* https://github.com/apple/foundationdb/pull/10808
+* https://github.com/apple/foundationdb/pull/10792
+* https://github.com/apple/foundationdb/pull/10803
+* https://github.com/apple/foundationdb/pull/10804
+* https://github.com/apple/foundationdb/pull/10824
+* https://github.com/apple/foundationdb/pull/10830
+* https://github.com/apple/foundationdb/pull/10832
+* https://github.com/apple/foundationdb/pull/10797
+* https://github.com/apple/foundationdb/pull/10789
+* https://github.com/apple/foundationdb/pull/10847
+* https://github.com/apple/foundationdb/pull/10860
+
+
+Earlier release notes
+---------------------
+
+* :doc:`71.3.0 (API Version 700) </release-notes/release-notes-7130>`
+* :doc:`71.3.1 (API Version 700) </release-notes/release-notes-7131>`
+* :doc:`71.3.2 (API Version 700) </release-notes/release-notes-7132>`
+* :doc:`71.3.4 (API Version 700) </release-notes/release-notes-7134>`
```

Comment:
Ordinarily these would be listed in reverse order (newest first) and would include all of the release notes going back forever. It's perhaps not a huge deal here, though, if we don't include all of the old ones for now.

Additionally, 71.3 has API version 710300

```suggestion
* :doc:`71.3.0 (API Version 700) </release-notes/release-notes-7130>`
* :doc:`71.3.1 (API Version 700) </release-notes/release-notes-7131>`
* :doc:`71.3.2 (API Version 700) </release-notes/release-notes-7132>`
* :doc:`71.3.4 (API Version 700) </release-notes/release-notes-7134>`
```

Reply:
Thanks for pointing out, seems like reversed here. Fixing it now

Reply:
> Additionally, 71.3 has API version 710300

Ah! Need to fix past release notes as well. My understanding was we update API version only for major releases, I remember confirming it with Ankita when wrote the first version. 
Fixing it now.